service/src/main/resources/schema/development/schema.json

```diff
@@ -180,6 +191,19 @@
           "type": "integer"
         }
       }
+    },
+    "samples": {
+      "type": "object",
+      "properties": {
+        "disease": {
```

Comment:
I think we need dictionaries for these too. We don't need necessarily need them for this checkin but we'd want to start the process so we can have a list in the future.